catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -84,6 +87,32 @@
     private Map<ServiceReference, ContentMetadataExtractor> contentMetadataExtractors =
             Collections.synchronizedMap(new TreeMap<>(new ServiceComparator()));
 
+    private MetacardType metacardType;
+
+    private boolean isMetadataTitleTheMetacardTitle;
+
+    /**
+     * @param metacardType                    must be non-null
+     * @param isMetadataTitleTheMetacardTitle must be non-null
+     */
+    public PdfInputTransformer(MetacardType metacardType, Boolean isMetadataTitleTheMetacardTitle) {
+        notNull(metacardType, "metacardType must be non-null");
+        notNull(isMetadataTitleTheMetacardTitle,
+                "isMetadataTitleTheMetacardTitle must be non-null");
+
+        this.metacardType = metacardType;
+        this.isMetadataTitleTheMetacardTitle = isMetadataTitleTheMetacardTitle;
+    }
+
+    /**
+     * @param metadataTitleTheMetacardTitle must be non-null
+     */
+    public void setIsMetadataTitleTheMetacardTitle(Boolean metadataTitleTheMetacardTitle) {
```

Comment:
I could be missing it, but I don't think I see unit test coverage for this option. Personally, I strongly prefer to see full branch coverage for user interaction points like option toggles.